lib/plenario_aot/aot_meta.ex

```diff
@@ -0,0 +1,45 @@
+defmodule PlenarioAot.AotMeta do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  alias PlenarioAot.AotMeta
+
+  schema "aot_metas" do
+    field :network_name, :string
+    field :slug, :string
+    field :source_url, :string
+    field :bbox, Geo.Polygon
+    field :time_range, Plenario.TsTzRange
+    timestamps()
+  end
+
+  def changeset(meta \\ %AotMeta{}, params \\ []) do
+    case is_map(params) do
+      true -> do_changeset(meta, params)
+      false -> do_changeset(meta, Enum.into(params, %{}))
+    end
+  end
+
+  defp do_changeset(meta, params) do
+    meta
+    |> cast(params, [:network_name, :source_url, :bbox, :time_range])
+    |> validate_required([:network_name, :source_url])
+    |> unique_constraint(:network_name)
+    |> unique_constraint(:source_url)
+    |> put_slug()
+  end
+
+  defp extract_non_ascii(string) do
```

Comment:
This is used to preserve non-ascii characters from being butchered by Slugify. It uses some library under to hood to try to transliterate the foreign chars, and frankly it's awful and unnecessary. URLs can handle UTF8, phoenix can handle it, so why mess with it?